packages/zowe-explorer-api/src/extend/MainframeInteraction.ts

```diff
@@ -71,7 +71,7 @@ export namespace MainframeInteraction {
          *
          * @param {imperative.Session} session a Zowe CLI Session
          */
-        logout?(session: imperative.Session);
+        logout?(session: imperative.Session): Promise<void>;
```

Comment:
This is a breaking change - although it isn't a huge change, it's probably worth adding to the changelog 😋